src/utils/FixedQueue.ts

```diff
@@ -0,0 +1,55 @@
+import * as _ from 'underscore';
```

Comment:
I would not because of a principle called "Tell don't ask". You might be interested by this Article: https://martinfowler.com/bliki/TellDontAsk.html

I want my data structure to handle all the logic that it needs to do instead of having a plain object that just contains data. The code is way easier to read that way!